tests/lib/parse.js

```diff
@@ -40,9 +40,6 @@ describe('parse()', () => {
   describe('general', () => {
     const code = 'let foo = bar;';
     const config = {
-      ecmaFeatures: {
-        blockBindings: true
```

Comment:
This was removed from ESLint in v2.0.0: https://eslint.org/docs/user-guide/migrating-to-2.0.0#language-options

Reply:
Good spot